packages/web-components/src/components/leaving-ibm/leaving-ibm.scss

```diff
@@ -29,9 +29,16 @@
   @extend :host(#{$prefix}-modal-body);
 
   padding-right: 20%;
+
+  ::slotted(slot[name='supplemental']) {
+    display: block;
+    margin-top: $spacing-07;
+    color: $text-02;
+    @include carbon--type-style('label-01');
+  }
 }
 
-:host(#{$dds-prefix}-leaving-ibm-modal-supplemental) {
+::slotted([slot='supplemental']) {
```

Comment:
would this affect other slots named 'supplemental'?

Reply:
ah good point. I can change it to target the classname

Reply:
@annawen1 @kennylam updated